charts/besu/README.md

```diff
@@ -30,7 +30,7 @@ An Ethereum execution layer client designed to be enterprise-friendly for both p
 | httpPort | int | `8545` | HTTP Port |
 | image.pullPolicy | string | `"IfNotPresent"` | besu container pull policy |
 | image.repository | string | `"hyperledger/besu"` | besu container image repository |
-| image.tag | string | `"22.10.3"` | besu container image tag |
+| image.tag | string | `"23.1.0-beta"` | besu container image tag |
```

Comment:
besu current image is quite broken. This is why I want to include a beta release until they fix main.